src/epub/text/chapter-8.xhtml

```diff
@@ -47,7 +47,7 @@
 			<p>“Just upon five o’clock, sir.”</p>
 			<p>“And was I⁠—er⁠—how was I?”</p>
 			<p>“Just a little cheerful, sir⁠—nothing more. Inclined to sing ‘<span epub:type="se:name.music.song">Rule Britannia</span>.’ ”</p>
-			<p>“What an extraordinary thing,” said Jimmy. “ ‘<span epub:type="se:name.music.song">Rule Britannia</span>,’ eh? I cannot imagine myself in a sober state ever singing ‘<span epub:type="se:name.music.song">Rule Britannia</span>.’ Some latent patriotism must have emerged under the stimulus of⁠—er⁠—just a couple too many. I was celebrating at the Mustard and Cress, I remember. Not nearly such an innocent spot as it sounds, Stevens.” He paused. “I was wondering⁠—”</p>
+			<p>“What an extraordinary thing,” said Jimmy. “⁠‘<span epub:type="se:name.music.song">Rule Britannia</span>,’ eh? I cannot imagine myself in a sober state ever singing ‘<span epub:type="se:name.music.song">Rule Britannia</span>.’ Some latent patriotism must have emerged under the stimulus of⁠—er⁠—just a couple too many. I was celebrating at the Mustard and Cress, I remember. Not nearly such an innocent spot as it sounds, Stevens.” He paused. “I was wondering⁠—”</p>
```

Comment:
This is still incorrect,  there must be a hair space between the quotes at `said Jimmy. “⁠‘<span`. This line should have no changes.

Reply:
Reverted. Sorry, I thought I had already done that.